### DISCUSSION TITLE
Better visibility in battery save mode

### DISCUSSION THREAD

**Discussion by Pyrdacor** [gsantner/markor#1804] [Ideas] [answered]
On Android 11 when using the battery save mode, the text color changes from black to white. As the background of the folder/files view is light gray, the names of the files and folders are very hard to read. Maybe the app should react to the battery save mode or default font color change, or just use the color black in any case.

**Comment by gsantner** [comment, marked as answer]
You may also just set the Black/AMOLED theme in Markor settings.

**Reply by Pyrdacor** [reply]
Then it's the other way around. The text color becomes black when leaving the battery save mode and you can't see the names either. I think something is broken with the text colors in general. The text color is not directly changed after I change the battery save mode but after a short while when navigating or open and close files.

**Reply by Pyrdacor** [reply]
Btw this only happens recently. Either it's due to the recent OS update (Fairphone 3 update to Android 11) or because of the recent update of the markdown viewer.

**Reply by gsantner** [reply]
I cannot reproduce

[screenrecord-2022-08-21_00.10.04.webm](https://user-images.githubusercontent.com/6735650/185767703-cfff8ad3-05ba-4a71-8520-eece530e2895.webm)

**Reply by Pyrdacor** [reply]
I could reproduce only in the following way:

- Set theme to "Bright" and activate save battery mode (or "Dark" and turn it off)
- Then close the app and start it again
- Open a directory which contains at least one file
- Open the file
- Close the file with the "Back" button of the OS

Then the filenames and foldernames have nearly the same color as the background. It is not reproducible without restarting the app.

**Reply by Pyrdacor** [reply]
https://user-images.githubusercontent.com/8607784/185768952-c143caa4-48d5-4670-9160-358d7eab713d.mp4

**Reply by Pyrdacor** [reply]
https://user-images.githubusercontent.com/8607784/185769011-a64bde76-8a19-4c30-85f5-a1107725b1fb.mp4
